lotto.py

```
@@ -0,0 +1,692 @@
+
+
+def _sign_anchor(anchor_hash: str) -> tuple[str, str]:
+    pubkey_b64, private_key = _load_operator_keypair()
+    signature = private_key.sign(bytes.fromhex(anchor_hash))
+    return pubkey_b64, base64.b64encode(signature).decode("ascii")
+
+
+def _fetch_drand_randomness(round_target: int) -> str:
+    url = settings.lotto_drand_url_template.format(round=round_target)
+    try:
+        with urllib.request.urlopen(url, timeout=10) as response:
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)